doc/doc_comments.mld

```diff
@@ -0,0 +1,42 @@
+{0 Doc-comments language reference}
+
+OCamlFormat uses {{:https://github.com/ocaml/odoc}odoc} to parse doc-comments (also referred to as doc-strings), and hence it inherits the accepted language from [odoc] (detailed in {{:https://ocaml.github.io/odoc/odoc_for_authors.html}odoc's documentation}).
+
+Here is an example showing a few useful elements:
+
+{%html:
+<pre>
+<code class="ml hljs ocaml">(** {0 Adding integers} *)
```

Comment:
`{0 }` is only possible for pages, not signatures because the rendered documentation will have the equivalent of `{0 module Foo}` already at the top. The most reasonable thing to write here is a synopsis: a paragraph explaining the module:

```suggestion
<code class="ml hljs ocaml">(** Adding integers. *)
```